apps/web/src/hooks/useNativeLoginResult.ts

```
@@ -0,0 +1,29 @@
+      try {
+        const message = JSON.parse(event.data);
+
+        if (message.type === WEBBRIDGE_MESSAGE_TYPE.SOCIAL_LOGIN_SUCCESS) {
+          const { accessToken, refreshToken } = message.payload;
+          setCookie('access_token', accessToken);
+          setCookie('refresh_token', refreshToken);
+          router.replace('/home');
+        } else if (message.type === WEBBRIDGE_MESSAGE_TYPE.SOCIAL_LOGIN_ERROR) {
+          router.replace('/login');
+        }
+      } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)